githubutils/githubutils_test.go

```diff
@@ -0,0 +1,48 @@
+package githubutils
+
+import (
+	"context"
+	"github.com/google/go-github/github"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/solo-io/go-utils/versionutils"
+)
+
+var _ = Describe("github utils", func() {
+	var (
+		client   *github.Client
+		ctx      = context.Background()
+		owner    = "solo-io"
+		reponame = "testrepo"
+		version = "v0.0.16"
+		ref = "master"
```

Comment:
Might make sense to use ref = "v0.0.16" too or some commit hash, don't want the test to break because master changes in the future

Reply:
Also gofmt